CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Gatekeeper
 Authorization client and server for tidepool
 
+## 0.4.3 - 2020-10-26
```

Comment:
Missing bump in package.json & package-lock.json

Reply:
reviewed